manifests/t/th-ch/YouTubeMusic/1.18.0/th-ch.YouTubeMusic.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: th-ch.YouTubeMusic
+PackageVersion: 1.18.0
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft

```

Comment:
```suggestion
  InstallerType: portable
```
![image](https://user-images.githubusercontent.com/11600822/193875462-94ad75c1-6902-4139-b0d4-86841bece79f.png)

`.EXE` files without `-Setup` is a portable executable file. Looks like the publisher didn't publish proper installers for this version so you'll need to change `InstallerType` to be `portable`